### DISCUSSION TITLE
content not load sometimes

### DISCUSSION THREAD

**Discussion by Abbasjafri-syed** [solodit/feedback#35] [Bug Report]
I usually face problem of content loading either it load too slow or just endsup a blank white page.
I think its load server issue as solodit is used by many individuals that can impact t loading ability which needed to be upgraded.
N.B
all other website are working properly and loading and not my net issue..

**Comment by hans-cyfrin** [comment]
Please let us know if this problem still persists.

**Comment by ghost** [comment]
@hans-cyfrin, I was reading CodeHawks documentation and in their [docs](https://docs.codehawks.com/hawks-auditors/how-to-evaluate-a-finding-severity#low-severity-findings) there's is a link to an example of a low finding on Solodit which doesn't load.

This is the link of the finding: 

- https://solodit.xyz/issues/l-06-erc1155action-returns-false-on-supportsinterface-with-the-real-erc1155-interface-code4rena-notional-notional-contest-git

**Reply by hans-cyfrin** [reply]
Thanks, we will check and update you here.
